package.json

```diff
@@ -8,15 +8,15 @@
 		"test": "run-s lint:* build",
 		"build": "webpack --mode=production",
 		"watch": "webpack --mode=development --watch",
-		"prerelease:version": "VERSION=$(utc-version); echo $VERSION; dot-json distribution/manifest.json version $VERSION",
-		"prerelease:source-url": "if [ ! -z \"${TRAVIS_REPO_SLUG}\" ]; then echo https://github.com/$TRAVIS_REPO_SLUG/tree/\"${TRAVIS_TAG:-$TRAVIS_COMMIT}\" > distribution/SOURCE_URL; fi",
-		"release": "npm-run-all build prerelease:* release:*",
+		"version": "dot-json distribution/manifest.json version $VER",
 		"release:cws": "webstore upload --source=distribution --auto-publish",
-		"release:amo": "web-ext-submit --source-dir distribution"
+		"release:amo": "web-ext-submit --source-dir distribution",
+		"release": "VER=$(daily-version) run-s build version release:*"
```

Comment:
The last improvement I'd make would be to let the user optionally set the version:

```sh
npm run release # uses daily-version
npm run release -- 20.1.1 # uses 20.1.1 as version
```

Suggestions welcome for this, but it can also come in the future.

Reply:
How would the user specify the version, modify package.json in a commit?

Reply:
This is similar to manually creating a hotfix tag right, or am I missing something?

Reply:
> How would the user specify the version, modify package.json in a commit?

Currently we never save the version in the repo itself, we just create tags. The use would specify the version when releasing, as specified: 
```js
npm run release -- 20.1.1 # uses 20.1.1 as version
```

> This is similar to manually creating a hotfix tag right, or am I missing something?

Kinda, yeah, but `hotfix` is just a way to trigger GHA; if you want to deploy from your computer, you can just run the `release` npm script. 

Currently in Refined GitHub there's no easy way to deploy from your computer because the scripts are tightly integrated with Travis. These latest changes allow you to release from your computer by running

```
npm run release
```

Reply:
Something like `VER=$([ -z $1 ] && daily-version || echo $1)` should let us do `npm run release 20.1.1`, right?

Reply:
That's not how npm script works, you can only pass arguments after `--`